services/gateway/apis/http-version3/swagger/definitions/transactions.json

```diff
@@ -10,60 +10,162 @@
-                            "type": "integer",
-                            "description": "Count of all transactions in the given time span",
-                            "example": "14447177193385"
-                        },
-                        "volume": {
-                            "type": "string",
-                            "description": "Sum of all transaction amounts and fees sent in the given time span (in Beddows)",
-                            "example": "14447177193385"
-                        }
+            "distributionByType": {
+                "type": "object",
+                "description": "Count of transactions by transaction type in the whole time interval",
+                "properties": {
+                    "token:transfer": {
+                        "type": "integer",
+                        "example": 1
                     }
                 }
             },
-            "distributionByType": {
+            "distributionByAmount": {
                 "type": "object",
-                "description": "Count of transactions by transaction type in the whole time interval"
+                "description": "Count of transactions by amount range in the whole time interval",
+                "properties": {
+                    "0000000100000000": {
+                        "type": "object",
+                        "properties": {
+                            "1_10": {
+                                "type": "integer",
+                                "example": 1
+                            }
+                        }
+                    }
+                }
             },
-            "distributionByAmount": {
+            "timeline": {
                 "type": "object",
-                "description": "Count of transactions by amount range in the whole time interval"
+                "properties": {
+                    "0000000100000000": {
+                        "type": "array",
+                        "items": {
+                            "type": "object",
+                            "required": [
+                                "timestamp",
+                                "date",
+                                "transactionCount",
+                                "volume"
+                            ],
+                            "properties": {
+                                "timestamp": {
+                                    "type": "integer",
+                                    "description": "UNIX timestamp representing the beginning of the aggregation interval",
+                                    "example": 1556100060
+                                },
+                                "date": {
+                                    "type": "string",
+                                    "description": "Date representing the beginning of the aggregation interval",
+                                    "example": "2019-11-27"
+                                },
```

Comment:
Perhaps this could also be renamed to `duration` to keep it generic as suggested for the response meta.